README.md

```diff
@@ -10,7 +10,10 @@ Instead of manually calling `use()` for every request, `use()` is called automat
 
 ```js
 
-var superagent = require('superagent-use');
+var superagent = require('superagent');
+/* Pass the superagent object to superagent-use
+   in order to be provided with new `use` functionality */
+require('superagent-use')(superagent);
```

Comment:
> I don't think it makes any difference whether we use `object-assign` or `extend`...

Yes, but because superagent already has `extend` as a dependency, it doesn't really matter if we add it too.